apps/src/templates/testPages/WebLabTest.jsx

```diff
@@ -98,80 +162,81 @@ class WebLabTest extends Component {
-              <li>Use a student computer</li>
-              <li>Log into the computer with a student account</li>
-              <br />
-              <p>
-                Note: You do not need to be logged in to Code.org for this test
-                to be successful.
-              </p>
-              <label htmlFor="connect">
-                Click "Connect" to check whether your school's network supports
-                the Web Lab tool
-              </label>
-              <div id="status-table">
-                <table>
-                  <tbody>
-                    <tr>
-                      <th>Check</th>
-                      <th>Status</th>
-                    </tr>
-                    <tr>
-                      <td>
-                        Connected to
-                        <code>https://downloads.computinginthecore.org</code>
-                      </td>
-                      <td className="computinginthecore">
-                        {this.renderTestStatus(STATUS_COMPUTING_IN_THE_CORE)}
-                      </td>
-                    </tr>
-                    <tr>
-                      <td>
-                        Connected to <code>https://codeprojects.org</code>
-                      </td>
-                      <td className="codeprojects">
-                        {this.renderTestStatus(STATUS_CODE_PROJECTS)}
-                      </td>
-                    </tr>
-                  </tbody>
-                </table>
-              </div>
-              <div>
-                <button
-                  type="submit"
-                  id="connect"
-                  className="btn btn-primary"
-                  onClick={this.runWebLabTest}
-                >
-                  Connect
-                </button>
-              </div>
-              {this.state.renderCallToAction && this.renderCallToAction()}
-              <br />
-            </div>
+      <div className={'contentContainer'}>
+        <div className={'content'} style={styles.content}>
+          <h1>Web Lab Network Support Check</h1>
+          <p>
+            Use this page to test your school's network support for our Web Lab
+            modules. Web Lab communicates with domains which may be blocked by
+            some school systems’ networks or device policies. If the network
+            check is unsuccessful, you will need to reach out to your school's
+            IT department to update your school's firewall settings or device
+            policies.
+          </p>
+          <p>
+            When checking network support, try to meet as many of these criteria
+            as possible:
+          </p>
+          <li>Use your school's internet connection</li>
+          <li>Use a student computer</li>
+          <li>Log into the computer with a student account</li>
+          <br />
+          <p>
+            Note: You do not need to be logged in to Code.org for this test to
+            be successful.
+          </p>
```

Comment:
It seems we should already know whether they are signed in or out, so we might be able to customize or not show this text in some cases?

Reply:
I could see a scenario where a logged-in teacher reads the instructions carefully on their account and then directs students/a class of students to perform the test. In that scenario, it might be useful for the teacher to know that the logged-in status does not matter, even though the teacher accessing the page is logged in. Does a scenario like that justify having the message about the logged-in state displayed all the time?

Reply:
Great question.  I think for now I'd opt for minimalism and not worry about mentioning it.  But it's not a strongly-held opinion.